neptune/sessions.py

```diff
@@ -61,9 +61,31 @@ def __init__(self, api_token=None, proxies=None):
         self._client = Client(self.credentials.api_address, self.credentials.api_token, proxies)
 
     def get_project(self, project_qualified_name):
-        """
+        """Get a project with given project qualified name.
+
+        In order to access experiment data one needs to get a `Project` object first.
```

Comment:
`Project` -> Add link to the class for example: ":class:`~neptune.projects.Project`" (and build docs locally to test that link works correctly)